quora-api/src/main/java/com/upgrad/quora/api/controller/QuestionController.java

```diff
@@ -52,7 +52,7 @@
    * @return List of QuestionDetailsResponse
    * @throws AuthorizationFailedException In case the access token is invalid.
    */
-  @RequestMapping(
+  @GetMapping(
       method = RequestMethod.GET,
```

Comment:
remove this line, same as @GetMapping annotation means method is mapped to HTTP get request